### PR TITLE
Incremental load logic

### PR DESCRIPTION
**Pull request by ondrajodas**
Solving - https://keboola.atlassian.net/browse/COM-24

### CODE REVIEW

File: src/Keboola/DbExtractor/Extractor/Firebird.php [hunk 1]
```diff
@@ -119,19 +175,32 @@ public function simpleQuery(array $table, array $columns = array()): string
     {
         if (count($columns) > 0) {
             $query = sprintf(
-                'SELECT %s FROM %s',
+                'SELECT %s%s FROM %s',
```
Comment (pivnicek) [inline src/Keboola/DbExtractor/Extractor/Firebird.php]: aha, this shocked me, thought it was a typo:).  maybe better to have `%s %s` and remove the space from `getIncrementalLimitAddon`?

File: src/Keboola/DbExtractor/Extractor/Firebird.php [hunk 2]
```diff
@@ -119,19 +175,32 @@ public function simpleQuery(array $table, array $columns = array()): string
...
                 implode(', ', array_map(function ($column): string {
                     return $column;
                 }, $columns)),
+                $this->getIncrementalLimitAddon(),
                 $table['tableName']
             );
         } else {
             $query = sprintf(
-                'SELECT * FROM %s',
+                'SELECT %s* FROM %s',
```
Comment (pivnicek) [inline src/Keboola/DbExtractor/Extractor/Firebird.php]: also here would be more readable with space

**Review by pivnicek — CHANGES_REQUESTED** [review]
skvely!  just a little change for readability please :-)

**Comment by ondrajodas** [comment]
@pivnicek ok i changed it and i removed configuration/tablesNode -> is no longer needed

**Review by pivnicek — APPROVED** [review]
ok 👍

**Pull request merged**